Changelog.md

```diff
@@ -175,337 +175,337 @@
+     * [BitBucket pull request 138](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/138)
 1. Added urdf gazebo extension option to disable fixed joint lumping
-     * [Pull request 133](https://bitbucket.org/osrf/sdformat/pull-request/133)
+     * [BitBucket pull request 133](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/133)
 1. Support urdfdom 0.3 (alternative to pull request #122)
-     * [Pull request 141](https://bitbucket.org/osrf/sdformat/pull-request/141)
+     * [BitBucket pull request 141](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/141)
 1. Update list of supported joint types
-     * [Pull request 142](https://bitbucket.org/osrf/sdformat/pull-request/142)
+     * [BitBucket pull request 142](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/142)
 1. Ignore unknown elements
-     * [Pull request 148](https://bitbucket.org/osrf/sdformat/pull-request/148)
+     * [BitBucket pull request 148](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/148)
 1. Physics preset attributes
-     * [Pull request 146](https://bitbucket.org/osrf/sdformat/pull-request/146)
+     * [BitBucket pull request 146](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/146)
 1. Backport fix for latin locales (pull request #147)
-     * [Pull request 150](https://bitbucket.org/osrf/sdformat/pull-request/150)
+     * [BitBucket pull request 150](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/150)
 
 ## SDFormat 1.4
 
 ### SDFormat 1.4.8 (2013-09-06)
 
 1. Fix inertia transformations when reducing fixed joints in URDF
-    * [Pull request 48](https://bitbucket.org/osrf/sdformat/pull-request/48/fix-for-issue-22-reducing-inertia-across/diff)
+    * [BitBucket pull request 48](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/48/fix-for-issue-22-reducing-inertia-across/diff)
 1. Add <use_terrain_paging> element to support terrain paging in gazebo
-    * [Pull request 47](https://bitbucket.org/osrf/sdformat/pull-request/47/add-element-inside-heightmap/diff)
+    * [BitBucket pull request 47](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/47/add-element-inside-heightmap/diff)
 1. Further reduce console output when using URDF models
-    * [Pull request 46](https://bitbucket.org/osrf/sdformat/pull-request/46/convert-a-few-more-sdfwarns-to-sdflog-fix/diff)
-    * [Commit](https://bitbucket.org/osrf/sdformat/commits/b15d5a1ecc57abee6691618d02d59bbc3d1b84dc)
+    * [BitBucket pull request 46](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/46/convert-a-few-more-sdfwarns-to-sdflog-fix/diff)
+    * [Commit](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/commits/b15d5a1ecc57abee6691618d02d59bbc3d1b84dc)
```

Comment:
This didn't work for me. Not sure how we map commit hashes.

Reply:
That's a valid link for me. From that page, you need to click on `View on GitHub` on the right.

Reply:
ah, when I clicked it from my editor, it added an extra parenthesis. Works fine.